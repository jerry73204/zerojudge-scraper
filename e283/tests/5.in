1
0 1 1 1
13
0 0 1 0
1 1 0 1
1 1 0 1
0 1 1 1
1 1 0 1
1 0 0 0
1 1 0 0
1 1 0 1
0 1 0 1
1 1 0 1
0 1 0 1
1 1 0 0
1 1 0 1
5
0 1 0 1
1 0 0 0
0 1 1 1
1 0 0 0
1 0 0 0
19
1 1 0 1
0 1 1 1
0 1 1 1
0 1 1 1
1 1 0 0
1 1 0 0
1 0 0 0
0 1 0 1
0 0 1 0
1 0 0 0
0 1 0 1
1 1 0 0
0 0 1 0
1 1 0 1
1 1 0 1
1 0 0 0
0 1 0 1
1 0 0 0
0 1 0 1
12
1 1 0 0
1 1 0 0
1 1 0 1
1 1 0 1
1 1 0 0
1 1 0 1
1 1 0 0
0 1 1 1
0 0 1 0
1 0 0 0
1 1 0 1
1 1 0 1
1
1 0 0 0
20
1 1 0 1
0 1 1 1
0 0 1 0
1 1 0 1
0 1 0 1
0 1 1 1
1 0 0 0
0 1 0 1
1 1 0 0
1 1 0 1
0 0 1 0
1 1 0 0
0 0 1 0
1 1 0 0
0 0 1 0
0 1 0 1
1 0 0 0
0 0 1 0
1 1 0 1
1 0 0 0
20
1 1 0 0
1 0 0 0
0 1 1 1
1 1 0 0
0 1 0 1
1 1 0 1
0 1 1 1
0 1 0 1
0 0 1 0
0 1 1 1
0 0 1 0
1 0 0 0
1 0 0 0
0 1 1 1
1 0 0 0
1 0 0 0
0 0 1 0
0 0 1 0
1 0 0 0
0 1 1 1
12
0 1 0 1
1 0 0 0
0 0 1 0
1 1 0 1
1 1 0 1
0 1 1 1
1 1 0 0
0 1 1 1
0 0 1 0
0 1 1 1
1 0 0 0
1 0 0 0
8
0 0 1 0
0 1 1 1
0 1 0 1
0 1 1 1
1 0 0 0
0 1 1 1
0 0 1 0
1 1 0 0
12
1 1 0 0
0 1 1 1
0 0 1 0
0 1 1 1
0 0 1 0
1 0 0 0
0 1 0 1
0 0 1 0
0 1 0 1
0 1 0 1
0 1 1 1
0 1 0 1
7
0 0 1 0
0 1 1 1
0 0 1 0
1 1 0 1
1 1 0 1
0 0 1 0
1 1 0 1
9
1 0 0 0
0 1 1 1
0 1 1 1
0 1 1 1
1 1 0 1
1 1 0 1
0 1 0 1
0 1 1 1
0 1 0 1
12
0 1 0 1
0 1 1 1
1 1 0 0
1 0 0 0
0 0 1 0
1 1 0 1
0 1 1 1
1 1 0 0
1 0 0 0
0 1 0 1
1 1 0 1
1 1 0 1
5
0 1 0 1
0 0 1 0
0 1 1 1
1 1 0 1
0 0 1 0
15
1 1 0 0
1 0 0 0
1 0 0 0
1 1 0 1
1 1 0 0
0 0 1 0
0 1 0 1
1 1 0 1
1 0 0 0
0 1 1 1
1 0 0 0
1 0 0 0
1 1 0 0
0 1 1 1
1 1 0 1
4
1 1 0 1
1 0 0 0
0 1 1 1
0 0 1 0
10
1 1 0 1
1 1 0 0
0 0 1 0
0 1 1 1
1 1 0 1
0 1 1 1
0 1 1 1
0 0 1 0
0 0 1 0
0 0 1 0
