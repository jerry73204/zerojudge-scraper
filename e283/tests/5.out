B
CDDBDEFDADAFD
AEBEE
DBBBFFEACEAFCDDEAEA
FFDDFDFBCEDD
E
DBCDABEAFDCFCFCAECDE
FEBFADBACBCEEBEECCEB
AECDDBFBCBEE
CBABEBCF
FBCBCEACAABA
CBCDDCD
EBBBDDABA
ABFECDBFEADD
ACBDC
FEEDFCADEBEEFBD
DEBC
DFCBDBBCCC